src/document_to_podcast/inference/model_loaders.py

```diff
@@ -28,62 +23,70 @@ def load_llama_cpp_model(
     model = Llama.from_pretrained(
         repo_id=f"{org}/{repo}",
         filename=filename,
-        # 0 means that the model limit will be used, instead of the default (512) or other hardcoded value
-        n_ctx=0,
+        n_ctx=0,  # 0 means that the model limit will be used, instead of the default (512) or other hardcoded value
```

Comment:
Are you using some sort of auto-formater beyond ruff? I see some changes like this and the import orders and wonder if we have a different pre-commit config

Reply:
Ruff wasnt working in Pycharm for me today (couldnt figure out why, something with the plugin) and I manually did that. I now installed the cli version and run it through there, so there shouldnt be any mismatch anymore.